pkg/oc/bootstrap/docker/up.go

```diff
@@ -47,6 +47,7 @@ import (
 	"github.com/openshift/origin/pkg/oc/bootstrap/docker/run"
 	"github.com/openshift/origin/pkg/oc/cli/util/clientcmd"
 	osclientcmd "github.com/openshift/origin/pkg/oc/cli/util/clientcmd"
+	"github.com/openshift/origin/pkg/version"
```

Comment:
@soltysh @juanvallejo you may want to create an oc version package if you think a split is likely.